Civi/WorkflowMessage/FieldSpec.php

```diff
@@ -16,6 +16,7 @@
 use Civi\Schema\Traits\PhpDataTypeSpecTrait;
 use Civi\Schema\Traits\OptionsSpecTrait;
 
+#[\AllowDynamicProperties]
```

Comment:
This class is new enough / has low enough usage we might be able to avoid this - where were the relevant test fails?

Reply:
```
1) Civi\WorkflowMessage\Traits\AddressingTraitTest::testFluentSetup
Creation of dynamic property Civi\WorkflowMessage\FieldSpec::$comment is deprecated

/home/seamus/buildkit/build/dtest/web/sites/all/modules/civicrm/Civi/Schema/Traits/ArrayFormatTrait.php:70
/home/seamus/buildkit/build/dtest/web/sites/all/modules/civicrm/Civi/WorkflowMessage/Traits/ReflectiveWorkflowTrait.php:66
/home/seamus/buildkit/build/dtest/web/sites/all/modules/civicrm/Civi/WorkflowMessage/Traits/ReflectiveWorkflowTrait.php:82
/home/seamus/buildkit/build/dtest/web/sites/all/modules/civicrm/Civi/WorkflowMessage/Traits/ReflectiveWorkflowTrait.php:137
/home/seamus/buildkit/build/dtest/web/sites/all/modules/civicrm/Civi/WorkflowMessage/WorkflowMessage.php:116
/home/seamus/buildkit/build/dtest/web/sites/all/modules/civicrm/Civi/WorkflowMessage/GenericWorkflowMessage.php:60
```

but also Civi\WorkflowMessage\ExampleWorkflowMessageTest::testValidateFail and Civi\WorkflowMessage\ExampleWorkflowMessageTest::testImpromptuImportExport failed in the same way